### ISSUE TITLE
Allow execution of the ETS via WebGUI and REST interface

### ISSUE THREAD

**Issue by lgoltz** [opengeospatial/ets-swecommon20#5]
and improve documentation of both interfaces.

Currently the main page of the WebGUI contains only the default informations. Running the test with [1] results in the following error 
```
Error in call to extension function {public java.lang.Object com.occamlab.te.TECore.callFunction(net.sf.saxon.expr.XPathContext,java.lang.String,java.lang.String,net.sf.saxon.om.NodeInfo) throws java.lang.Exception}: Exception in extension function java.lang.Exception: Fatal Error: XPath syntax error at char 19 on line 47 in {tng:new($outputDir)}:
    Cannot find a matching 1-argument function named {java:org.opengis.cite.swecommon20.TestNGController}new(); SystemID: file:/root/te_base/work/_root_te_base_scripts_swecommon20_2.0_ctl_/swecommon20-suite/tns$run-ets-swecommon20.fn; Line#: 47; Column#: -1
Fatal Error: XPath syntax error at char 40 on line 50 in {...Run($controller, $testRunAr...}:
    Cannot find a matching 2-argument function named {java:org.opengis.cite.swecommon20.TestNGController}doTestRun(); SystemID: file:/root/te_base/work/_root_te_base_scripts_swecommon20_2.0_ctl_/swecommon20-suite/tns$run-ets-swecommon20.fn; Line#: 50; Column#: -1
Failed to compile stylesheet. 2 errors detected.
Result: Failed
```

REST interface was not tested yet.

[1]   https://raw.githubusercontent.com/opengeospatial/ets-swecommon20/master/src/test/resources/examples/image_data.xml

**Comment by lgoltz** [comment]
Enhanced teamengine-docker: https://github.com/opengeospatial/teamengine-docker/pull/18

**Comment by epicureanism** [comment]
bug fixed f46bb50472fb742413fca681b3689cf8628dca07 and also tested by Teamengine-docker.


![image](https://user-images.githubusercontent.com/1300018/47772084-85e6fd80-dd20-11e8-99dd-97b8526c09b3.png)
